Library/Formula/redshift.rb

```diff
@@ -0,0 +1,74 @@
+class Redshift < Formula
+  homepage "http://jonls.dk/redshift/"
+  url "https://github.com/jonls/redshift/releases/download/v1.10/redshift-1.10.tar.xz"
+  sha1 "7b06376ad96305252b93fa2ef14af47b521c82ab"
+
+  head do
+    url "https://github.com/jonls/redshift.git"
+
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "intltool" => :build
+  depends_on "gettext"
+  depends_on "glib"
+
+  def install
+    # Despite the naming, 'quartz' here does not = x11 dep.
+    args = %W[
+      --prefix=#{prefix}
+      --enable-corelocation
+      --disable-silent-rules
+      --disable-dependency-tracking
+      --disable-geoclue
+      --disable-geoclue2
+      --enable-quartz
+      --with-systemduserunitdir=no
+      --disable-gui
+    ]
+
+    system "./bootstrap" if build.head?
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  plist_options :manual => "redshift"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{bin}/redshift</string>
+        </array>
+        <key>KeepAlive</key>
+        <true/>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>StandardErrorPath</key>
+        <string>/dev/null</string>
+        <key>StandardOutPath</key>
+        <string>/dev/null</string>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  def caveats; <<-EOS.undent
+    A .conf file has not been provided. If you want one, see:
+      http://jonls.dk/redshift/
+    And place it in $HOME/.config
+    EOS
+  end
+
+  test do
+    system "#{bin}/redshift", "-V"
```

Comment:
Not really. I could set ` redshift ` running, but unless you want me to manually explore the process list running and look for ` redshift `, verify it's running fine in that process list, and then kill it after the test, there's not a lot else we can do. Redshift's options are basically ` -V `, ` -h ` or ` redshift `, the latter of which just starts the binary and tints the screen as desired. There's very little in the way of *play with this option* settings. I'll double check for you, but I'm pretty sure that's the case.